audit/audit-s3-storage.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+import os
+import boto3
+from botocore.exceptions import ClientError
+import datetime
+import functools
+import json
+import subprocess
+
+s3_client = boto3.client("s3")
+cloudwatch_client = boto3.client("cloudwatch")
+
+PROD_S3_PREFIX = "cg-"
```

Comment:
```suggestion
PROD_S3_PREFIX = "cg-"
```

I think the prod s3 prefix is something like cg-broker-prd?

Reply:
I wish, its just cg-

Reply:
Ah , okay